recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,6 @@
+MACOSX_SDK_VERSION:
+  - '11.0'
 vc:
-  - 14
\ No newline at end of file
+  - 14
```

Comment:
By the way, the `vc: 14` can be dropped (and please add a trailing newline as is standard for git-based repos)